src/pusher.rs

```diff
@@ -128,6 +131,28 @@ impl ContainerPusher {
             ring::digest::digest(&ring::digest::SHA256, serialized_config.as_bytes()).as_ref(),
         );
         let content_length = serialized_config.len();
+
+        if let Some(_) = local_image_path {
+            let mut config_path = local_image_path.clone().unwrap();
+            config_path.push("blobs/sha256");
+            config_path.push(sha256_hash.clone());
+
```

Comment:
do we need to mkdir here to ensure it exists?

Reply:
Also, I think we should call this config.json

Reply:
No, we ensure the directory exists when pushing blobs